### ISSUE TITLE
toArray() should be detected as breaking out of the jQuery collection

### ISSUE THREAD

**Issue by stof** [wikimedia/eslint-plugin-no-jquery#96]
I have false positives for `no-jquery/no-map` or similar methods right now, for code like this:

```js
$col.toArray().map(someCallback)
```

In this case, `.map` is already a call to `Array.prototype.map`, not to `jQuery.map`, because `toArray()` returns an array, not a jQuery collection.

**Comment by stof** [comment]
btw, both `no-map` and `no-map-collection` are catching that call.
